artifactory/utils/dotnet/solution/solution.go

```diff
@@ -181,18 +202,19 @@ func (solution *solution) loadSingleProjectFromDir() error {
 
 	if len(projFiles) == 1 {
```

Comment:
We might want to log something here, because the cli will return without collecting build info and without throwing an error.